diff -aurN dunst-1.0.0.orig/Makefile dunst-1.0.0/Makefile
--- dunst-1.0.0.orig/Makefile	2013-08-10 12:55:02.417561338 +0200
+++ dunst-1.0.0/Makefile	2013-08-10 12:56:35.346433310 +0200
@@ -19,6 +19,7 @@
 options:
 	@echo dunst build options:
 	@echo "CFLAGS   = ${CFLAGS}"
+	@echo "LIBS     = ${LIBS}"
 	@echo "LDFLAGS  = ${LDFLAGS}"
 	@echo "CC       = ${CC}"
 
@@ -34,14 +35,14 @@
 
 dunst: ${OBJ}
 	@echo CC -o $@
-	@${CC} ${CFLAGS} -o $@ ${OBJ} ${LDFLAGS}
+	@${CC} ${LDFLAGS} ${CFLAGS} -o $@ ${OBJ} ${LIBS}
 
 dunstify:
 	@${CC} -o $@ dunstify.c -std=c99 $(shell pkg-config --libs --cflags glib-2.0 libnotify) 
 
 debug: ${OBJ}
 	@echo CC -o $@
-	@${CC} ${CFLAGS} -O0 -o dunst ${OBJ} ${LDFLAGS}
+	@${CC} ${LDFLAGS} ${CFLAGS} -O0 -o dunst ${OBJ} ${LIBS}
 
 clean:
 	@echo cleaning
diff -aurN dunst-1.0.0.orig/config.mk dunst-1.0.0/config.mk
--- dunst-1.0.0.orig/config.mk	2013-08-10 12:55:02.417561338 +0200
+++ dunst-1.0.0/config.mk	2013-08-10 12:55:29.312871170 +0200
@@ -35,7 +35,7 @@
 # includes and libs
 INCS := $(shell ${PKG_CONFIG} --cflags ${pkg_config_packs})
 CFLAGS += ${INCS}
-LDFLAGS += -lm -L${X11LIB} -lXss ${XINERAMALIBS} $(shell ${PKG_CONFIG} --libs ${pkg_config_packs})
+LIBS += -lm -L${X11LIB} -lXss ${XINERAMALIBS} $(shell ${PKG_CONFIG} --libs ${pkg_config_packs})
 
 # only make this an fatal error when where not cleaning
 ifneq (clean, $(MAKECMDGOALS))
